etc/ci/performance/firefox/revision_46.0.1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "commit": "0b8492c110be",
```

Comment:
Should this be a full (length 40) hash? Possibly related to https://bugzilla.mozilla.org/show_bug.cgi?id=1283856.